client/question/domain/SpeechBalloonObjectFactory.js

```diff
@@ -24,10 +24,14 @@ tie.factory('SpeechBalloonObjectFactory', [
     var SPEECH_BALLOON_TYPE_CODE = 'code';
     // A balloon representing feedback given by TIE.
     var SPEECH_BALLOON_TYPE_FEEDBACK = 'feedback';
+    // A balloon representing the intro message to be provided if the
+    // question is not shown (TIE is iframed).
+    var SPEECH_BALLOON_TYPE_INTRO = 'intro';
```

Comment:
What's the rationale for creating a separate type of balloon rather than using the existing feedback balloon? Probably worth documenting.

Reply:
Modified so that the public API distinguishes between intro, feedback, and code balloons, but intro balloons are actually inherently feedback balloons.